src/nvim/po/CMakeLists.txt

```diff
@@ -1,190 +1,190 @@
-find_package(Gettext REQUIRED)
-find_program(XGETTEXT_PRG xgettext)
-find_program(ICONV_PRG iconv)
-mark_as_advanced(
-  GETTEXT_MSGFMT_EXECUTABLE
-  GETTEXT_MSGMERGE_EXECUTABLE
-  ICONV_PRG
-  XGETTEXT_PRG)
-
-option(ENABLE_LANGUAGES "Localizations to build" ON)
-if(ENABLE_LANGUAGES)
-  set(LANGUAGES
-    af
-    ca
-    cs
-    da
-    de
-    en_GB
-    eo
-    es
-    fi
-    fr
-    ga
-    it
-    ja
-    ko.UTF-8
-    nl
-    no
-    pl.UTF-8
-    pt_BR
-    ru
-    sk
-    sv
-    tr
-    uk
-    vi
-    zh_CN.UTF-8
-    zh_TW.UTF-8)
-endif()
-
-if(HAVE_WORKING_LIBINTL AND GETTEXT_FOUND AND XGETTEXT_PRG AND ICONV_PRG)
-  set(ENV{OLD_PO_FILE_INPUT} yes)
-  set(ENV{OLD_PO_FILE_OUTPUT} yes)
-
-  set(NVIM_RELATIVE_SOURCES)
-  foreach(SRC ${NVIM_SOURCES} ${NVIM_HEADERS})
-    file(RELATIVE_PATH RELATIVE_SRC ${CMAKE_CURRENT_SOURCE_DIR} ${SRC})
-    list(APPEND NVIM_RELATIVE_SOURCES ${RELATIVE_SRC})
-  endforeach()
-
-  set(NVIM_POT ${CMAKE_CURRENT_BINARY_DIR}/nvim.pot)
-
-  list(SORT NVIM_RELATIVE_SOURCES)
-  add_custom_command(
-    OUTPUT ${NVIM_POT}
-    COMMAND $<TARGET_FILE:nvim_bin> -u NONE -i NONE -n --headless --cmd "set cpo+=+"
+option(ENABLE_TRANSLATIONS "Localizations to build" OFF)
+if(ENABLE_TRANSLATIONS)
```

Comment:
is there not a way to "return early" in cmake? :) that would avoid the churn in the rest of the file

Reply:
There is, but it's not documented and I just learned it by testing it out 🤪

Reply:
There is, but it's undocumented, so unsure it will behave as expected. We can test it out and see if any problems show up.